routes/helpers/groups.js

```diff
@@ -16,9 +16,14 @@ function groupEditPath(semester_name, module_code, group_name) {
   return `${groupPath(semester_name, module_code, group_name)}/edit`;
 }
 
+function groupmembershipPath(semester_name, module_code, group_name, user_id) {
```

Comment:
```suggestion
function groupMembershipPath(semester_name, module_code, group_name, user_id) {
```

pls use camel case for function names